bin/jobsub_fetchlog

```diff
@@ -51,6 +54,134 @@ from version import print_version, print_support_email
 
 verbose = 0
 
+# environment variable containing base fetchlog server url
+_FETCHLOG_URL_ENV = "JOBSUB_FETCHLOG_URL"
+# archive download chunk size in bytes
+_CHUNK_SIZE = 1024 * 1024
+
+
+def fetch_from_condor(
+    jobid: str, destdir: Optional[str], archive_format: str, partial: bool
+):
+    # find where the condor_transfer_data will put the output
+    j = Job(jobid)
+    iwd = j.get_attribute("SUBMIT_Iwd")
+    if verbose:
+        print(f"job output to {iwd}")
+    # make sure it exists, create if not
+    try:
+        os.stat(iwd)
+    except FileNotFoundError:
```

Comment:
That's not new though. Did it not work before?

Reply:
My impression is that if you submit from one machine, we create the cache dir there, and if you go to another machine to fetch the logs, condor doesn't see the IWD, so condor_transfer_data fails.  This should patch that issue, if I understand correctly.

Reply:
Right, but that code has been there since the first version of fetchlog https://github.com/marcmengel/jobsub_lite/commit/d43adaf2592982a8aa852b70c0bb3a35f37ebd89#diff-18845d9de59cabc77f13bd8167605da13f5edc1ae4ab6776ffc44b12c2f94ca4R96

Reply:
Ah strange.  Maybe I am getting it mixed up then.